server/config/db.js

```diff
@@ -0,0 +1,23 @@
+const {Pool} = require("pg");
+const fs = require("fs");
+
+
```

Comment:
You shouldn't add your password to source control - I'd recommend removing this, using a .env file, and changing your password.